client/blocks/nps-survey/index.jsx

```diff
@@ -155,13 +196,64 @@ class NpsSurvey extends Component {
 						className="nps-survey__not-answer-button"
 						onClick={ this.handleFeedbackFormClose }
 					>
-						{ translate( 'Close' ) }
+						{ this.shouldShowPromotion() ? translate( 'Skip' ) : translate( 'Close' ) }
 					</Button>
 				</div>
 			</Fragment>
 		);
 	}
 
+	renderPromotion() {
+		const { hasAvailableConciergeSession, translate } = this.props;
+
+		return (
+			<div className="nps-survey__promotion">
+				<p>{ translate( 'Thank you for your feedback. We’d like to help!' ) }</p>
+				{ hasAvailableConciergeSession && (
+					<Fragment>
+						<p>
+							{ translate(
+								'You have a free, 45-minute one-on-one call with a website expert as part of your WordPress.com Business plan benefits.'
+							) }
+						</p>
+						<p>
+							{ translate(
+								'{{booking}}Reserve a 1:1 Support Session{{/booking}} now or connect with the support team {{contact}}over live chat or email{{/contact}}.',
```

Comment:
I agree it could use to be more user-focused and benefit oriented and it's something we can definitely focus on for future iterations. Let's keep it as-is for now and we can adjust accordingly once we see how users engage with this version.

Reply:
> Let's keep it as-is for now and we can adjust accordingly once we see how users engage with this version.

👍